dynamodb.go

```diff
@@ -0,0 +1,149 @@
+package limiters
+
+import (
+	"context"
+	"sync"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/dynamodb"
+	"github.com/aws/aws-sdk-go-v2/service/dynamodb/types"
+	"github.com/pkg/errors"
+)
+
+// DynamoDBTableProperties are supplied to DynamoDB limiter backends
+type DynamoDBTableProperties struct {
+	TableName        string
+	PartitionKeyName string
+	SortKeyName      string
+	SortKeyUsed      bool
+	TTLFieldName     string
+	TTLUsed          bool
+}
+
+var tableCache map[string]DynamoDBTableProperties = make(map[string]DynamoDBTableProperties)
+var lock *sync.Mutex = &sync.Mutex{}
```

Comment:
Types can be omitted 

```suggestion
var tableCache = make(map[string]DynamoDBTableProperties)
var lock = &sync.Mutex{}
```